_locales/en/messages.json

```diff
@@ -428,6 +428,24 @@
     "selectAContact": {
       "message": "Select a contact or group to start chatting."
     },
+    "replyingToYourself": {
+      "message": "Replying to Yourself",
+      "description": "Shown in iOS theme when you quote yourself"
+    },
+    "replyingToYou": {
+      "message": "Replying to You",
+      "description": "Shown in iOS theme when someone else quotes a message from you"
+    },
+    "replyingTo": {
+      "message": "Replying to $name`$",
```

Comment:
What is the meaning of the backtick after `name`?

Reply:
Typo. :0)